languages/ruby/exercises/concept/floating-point-numbers/.docs/instructions.md

```diff
@@ -0,0 +1,41 @@
+In this exercise you'll be working with savings accounts. Each year, the balance of your savings account is updated based on its interest rate. The interest rate your bank gives you depends on the amount of money in your account (its balance):
```

Comment:
We will be working with savings accounts.  But it doesn't state "your savings accounts".  Should we refer to them as "the savings accounts" rather than "your"?  The same your bank and your account.  With the ownership tag, it may be limiting to only your account, rather than every account, or any account.